core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -160,6 +164,13 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
     }
   }
 
+  setFilters(){
+    // we are targeting the second object in the FormFields where values for AGGREGATION is there
+    this.formFields[2].options = ['Average', 'Total', 'Maximum']
```

Comment:
Is this accurate? We are seeing mixed case here. Do we need to set both value and values?

Reply:
value and values both are necessary as the filter modal component being used there needs both of these fields